### ISSUE TITLE
Add Arch PKGBUILD

### ISSUE THREAD

**Issue by Stazed** [Stazed/rakarrack-plus#6]
/arch/package/PKGBUILD

**Comment by jujudusud** [comment]
Hi here, is it on the way ?

**Comment by Stazed** [comment]
It has been a while since I did one of these.
Perhaps @ronjouch could send me the one he had for the old repository and save me some time :).
Otherwise, it might be a while.

**Comment by ronjouch** [comment]
> Perhaps @ronjouch could send me the one he had for the old repository and save me some time :).

@Stazed @jujudusud sure, here's what worked for me in January 2020:

```
# Maintainer: Your name here <your@email.here>
pkgname=rakarrack-lv2-git
pkgver=0.7.1.12.gc0c75e3
pkgrel=1
pkgdesc="Stereo digital effects processor with Jack support (continuation of ssj71's LV2 fork by Stazed with fixes & more, git version)"
arch=('i686' 'x86_64')
url="https://github.com/Stazed/rakarrack"
license=('GPL3')
depends=('alsa-utils' 'fftw' 'fltk' 'jack' 'libxpm')
makedepends=('git')
conflicts=('rakarrack')
provides=('rakarrack')
_name='rakarrack'
source=("git+https://github.com/Stazed/$_name.git#branch=wip")
sha256sums=('SKIP')

pkgver() {
  cd "$srcdir/$_name"
  git describe --long --tags | sed 's/Release-//;s/-/./g'
}

build(){
  mkdir -p "$srcdir/build"
  cd "$srcdir/build"
  cmake "$srcdir/$_name" \
        -DCMAKE_BUILD_TYPE=Release \
        -DCMAKE_INSTALL_PREFIX="/usr"
  make
}

package() {
  cd "$srcdir/build"
  make DESTDIR="$pkgdir" install
}
```

**Comment by jujudusud** [comment]
Ok, really nice.
I will test and update this one in order to create and maintain an AUR package.
Thank you guys !

**Comment by Rippert** [comment]
Hey, I have just built Rakarrack-Plus on an ARM machine. Raspberry Pi 4B 2G running latest Raspberry Pi OS (Debian Buster). Just followed the instructions from here, nothing special. One of my pet peeves is that almost all AUR packages set the "arch" to 'i686' 'x86_64', yet build fine on ARM. I haven't actually used ARCH on my Pi in a while, but I could set it up and test your package if that would help you make it easier for us ARM folks to use AUR. I know the arch can be overridden, but like I said, it's kind of a pet peeve of mine.

**Comment by Stazed** [comment]
There is now a working updated PKGBUILD in the arch folder. Currently in the wip branch. Will be included in the next major release 1.1.0 coming soon.
@ronjouch  thanks!!!
@Rippert send me a patch for the ARM fix if you want. Would like to get it in before the release, which will probably be mid next week.
Closing this.

**Comment by Rippert** [comment]
I just tested the new PKGBUILD file and it worked fine on a Raspberry Pi running Manjaro. There was only one edit:

line 6 of your file:

`arch=('i686' 'x86_64')`

becomes:

`arch=('i686' 'x86_64' 'aarch64')`
